webevd/WebEVD/web/evd.js

```diff
@@ -564,20 +564,39 @@ for(let label in spacepoints){
 }
 
 
-let colors = ['red', 'blue', 'green', 'orange', 'purple', 'skyblue'];
+let other_colours = ['green', 'purple', 'pink', 'red', 'violet']
+let pdg_colours = {'13': 'blue', '11': 'skyblue', '2212': 'orange', '211': 'magenta'}
 
-function add_tracks(trajs, group){
+function add_tracks(trajs, group, must_be_charged){
     let i = 0;
     for(let track of trajs){
-        let col = colors[i%colors.length];
+        let col = 'white';
+
+        if (('pdg' in track) && (track.pdg in pdg_colours))
+            col = pdg_colours[track.pdg];
+        else if (must_be_charged)
```

Comment:
It seems like it might be safer to blacklist neutral particles, rather than trying to whitelist charged ones.

Could we make neutral particles gray or something, when they're not suppressed, rather than random?